demos/fonts/all.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <style>
+      html {
+        font: 16px 'Helvetica Neue', Arial, sans-serif;
+      }
+      body {
+        margin: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <p>This demo renders three staves, each with a different music engraving font.</p>
+    <h3>Bravura</h3>
+    <div id="outputBravura"></div>
+    <h3>Gonville</h3>
+    <div id="outputGonville"></div>
+    <h3>Petaluma</h3>
+    <div id="outputPetaluma"></div>
+    <p>View the source code for more information.</p>
+    <!--
+        The standard VexFlow library (build/vexflow.js) includes three music fonts: Bravura, Gonville, and Petaluma.
+        It statically bundles all the music fonts, and selects Bravura as its default.
+        For this demo we include build/vexflow-debug.js, which is the non-minified version of vexflow.js.
+        The build/*.js files are compatible with <script> tags.
+        The build/esm/*.js files are compatible with ESM import statements.
+    -->
+    <script src="../../build/vexflow-debug.js"></script>
+    <script>
+      console.log(Vex);
+      console.log('VexFlow BUILD ' + Vex.Flow.BUILD);
+
+      {
+        // Calling setMusicFont() here is optional, since Bravura is the default font.
```

Comment:
I incorporated suggestions from @0xfe regarding additional comments in the demo files. I also removed the big horizontal separator comments. :-)